relayer/send_relayed_transaction.go

```diff
@@ -0,0 +1,84 @@
+package relayer
```

Comment:
I'd name this file `submit_transaction.go` or something like that - the transaction hasn't been relayed yet, so `send_relayed_transaction` is a bit weird lol

Reply:
The new name is `submit_transaction.go`, so I'll resolve this one.